core/variant/variant_conversion.h

```diff
@@ -0,0 +1,91 @@
+/*************************************************************************/
```

Comment:
I would probably keep this out of core if its only needed for your code, just put it locally.
The least stuff in core the better.